### ISSUE TITLE
Failure to Compile

### ISSUE THREAD

**Issue by maarek** [samcrow/xplm-sys#3]
Some bits have changed in the past 2 years and at least with Rustc 1.42.0, the build fails due to the bindgen dependency not able to be built. Looks like some of its dependencies have gone away too. What would be the effort to update the bindgen dependency to the most recent version?

```rust
   Compiling syntex_syntax v0.58.1
error[E0423]: expected function, tuple struct or tuple variant, found struct `ast::Name`
   --> /home/jeremy/.cargo/registry/src/github.com-1ecc6299db9ec823/syntex_syntax-0.58.1/src/symbol.rs:146:27
    |
146 |                       name: ast::Name($index),
    |                             ^^^^^^^^^
...
165 | / declare_keywords! {
166 | |     // Invalid identifier
167 | |     (0,  Invalid,        "")
168 | |
...   |
231 | |     (56, CrateRoot, "{{root}}")
232 | | }
    | |_- in this macro invocation

   Compiling env_logger v0.4.3
error: aborting due to previous error
```

**Comment by vanmellebenjamin** [comment]
Hi,

Did you find a solution ?

**Comment by samcrow** [comment]
Thanks for reporting this. I just released version 0.4.0, which does compile. It also no longer does the C - Rust translation at compile time, which should make things faster.
